sync/src/main/java/tech/pegasys/artemis/sync/SyncManager.java

```diff
@@ -128,9 +130,16 @@ synchronized boolean isSyncQueued() {
     return syncQueued;
   }
 
+  synchronized SyncStatus getSyncStatus() {
```

Comment:
I don't think this needs to be synchronized.  Especially once we're not storing `highestSlot` but as it stands the write isn't synchronized so it's not thread safe anyway.